Source/Core/VideoCommon/Present.cpp

```diff
@@ -23,13 +23,74 @@
 
 std::unique_ptr<VideoCommon::Presenter> g_presenter;
 
+// The video encoder needs the image to be a multiple of x samples.
```

Comment:
So is "LMC" least multiple count?  In your comment is x = 4?

Reply:
Perhaps this should be LCM ([least common multiple](https://en.wikipedia.org/wiki/Least_common_multiple)) instead?

Reply:
Damn i wrote it wrong, LCM yes

Reply:
Done